### PR TITLE
simplify_analysis-II

### PR DESCRIPTION
**Pull request by mschwoer**
simplify option loading for analysis page

### CODE REVIEW

**Review by boopthesnoot — APPROVED** [review]
good decluttering effort!

File: alphastats/gui/pages/04_Analysis.py [hunk 1]
```diff
@@ -1,12 +1,12 @@
 import streamlit as st
+from gui.utils.options import get_plotting_options, get_statistic_options
```
Comment (JuliaS92) [inline alphastats/gui/pages/04_Analysis.py:2]: I get that we wanted to declutter the session state. How expensive is it to run these functions? Maybe we can at least restrict that to once per page reload, or cache the result.

**Review by JuliaS92 — APPROVED** [review]
Since we wanted to prioritize function over performance now, my one comment is mostly for future consideration.

File: alphastats/gui/pages/04_Analysis.py (hunk 1 above)
Comment (mschwoer) [reply alphastats/gui/pages/04_Analysis.py:2]: they will be removed in a later PR

**Pull request merged**